Toolkit/ArcGISToolkit/TimeSlider.swift

```diff
@@ -1292,7 +1292,7 @@ public class TimeSlider: UIControl {
         // Add the minimum value label
         fullExtentStartTimeLabel.isHidden = !(fullExtentLabelsVisible && isSliderVisible)
         fullExtentStartTimeLabel.foregroundColor = fullExtentLabelColor.cgColor
-        fullExtentStartTimeLabel.alignmentMode = kCAAlignmentCenter
+        fullExtentStartTimeLabel.alignmentMode = CATextLayerAlignmentMode.center
```

Comment:
I would use implicit member expressions for all of these:

```suggestion
        fullExtentStartTimeLabel.alignmentMode = .center
```